pages/orders/[orderId]/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { useRouter } from 'next/router';
+import React from 'react';
+import styled from 'styled-components';
+
+import ErrorMessage from '../../../components/error';
+import Loading from '../../../components/loading';
+import { useShippingAndProductsOrder } from '../../../lib/hooks';
+import { BillingAddress, OrderProduct, ShippingAndProductsOrder } from '../../../types';
+
+const StyledAddress = styled.address`
+    color: ${({ theme }) => theme.colors.secondary70};
+    font-style: normal;
+    line-height: ${({ theme }) => theme.lineHeight.small};
+`;
+
+const StyledUl = styled.ul`
+    ${({ theme }) => theme.helpers.listReset}
+
+    color: ${({ theme }) => theme.colors.secondary70};
+    line-height: ${({ theme }) => theme.lineHeight.xxLarge};
+`;
+
+const InternalOrderPage = (order: ShippingAndProductsOrder) => {
+    const { shipping_addresses = [], products = [] } = order
+    const items = shipping_addresses.map(address => {
+        const addressProducts = products.filter(({ order_address_id }) => order_address_id === address.id);
+
+        return { address, addressProducts }
+    });
+
+    const Address = (address: BillingAddress) => (
+        <StyledAddress>
+            <div>{address.first_name} {address.last_name}</div>
+            <div>{address.street_1} {address.street_2}</div>
+            <div>{address.city}, {address.state} {address.zip}</div>
+        </StyledAddress>
+    );
+
+    const renderOrderProducts = (addressProducts: OrderProduct[]) => (
+        <StyledUl>
+            {addressProducts.map(product => <li key={product.id}>{product.name}</li>)}
+        </StyledUl>
+    );
+
+    const renderOrderProductsQuantities = (addressProducts: OrderProduct[]) => (
+        <StyledUl>
+            {addressProducts.map(product => <li key={product.id}>{product.quantity}</li>)}
+        </StyledUl>
+    );
+
+    return (
+        <>
+            <Flex justifyContent="space-between" marginBottom="medium">
+                <H1>Order details</H1>
+                <Button>Create shipment</Button>
+            </Flex>
+            <Panel>
+                <StatefulTable
+                    columns={[
+                        { header: 'Ship to', hash: 'address', render: ({ address }) => <Address {...address} /> },
+                        {
+                            header: 'Products',
+                            hash: 'addressProducts',
+                            render: ({ addressProducts }) => renderOrderProducts(addressProducts),
+                        },
+                        {
+                            header: 'Quantity',
+                            hash: 'quantity',
+                            render: ({ addressProducts }) => renderOrderProductsQuantities(addressProducts),
+                        },
+                    ]}
+                    items={items}
+                /> 
+            </Panel>
+        </>
+    );
+};
+
+const OrderPage: React.FC = () => {
```

Comment:
please remove `React.FC`, it's not necessary.
```suggestion
const OrderPage = () => {
```